### PR TITLE
Nar 295 quote handling

### PR DESCRIPTION
**Pull request by oskardrenske-ith**

### CODE REVIEW

File: tests/integration_tests/search/test_querybuilder.py [hunk 1]
```diff
@@ -34,29 +33,28 @@ def test_build_occupation_collection_query(collection_id, expected):
...
-@pytest.mark.parametrize("querystring, expected", [('xx,', 'xx '),  # trailing space
+@pytest.mark.parametrize("querystring, expected", [('xx,', 'xx'),  # trailing space
                                                    ('xx.', 'xx'),
                                                    ('xx!', 'xx'),
                                                    ('xx?', 'xx'),
                                                    ('xx:', 'xx'),
                                                    ('xx;', 'xx'),
                                                    ('.xx', 'xx'),
-                                                   (',xx', ' xx'),  # leading space
+                                                   (',xx', 'xx'),  # leading space
```
Comment (halge) [inline tests/integration_tests/search/test_querybuilder.py:43]: # leading space without space?

File: tests/integration_tests/search/test_querybuilder.py [hunk 2]
```diff
@@ -34,29 +33,28 @@ def test_build_occupation_collection_query(collection_id, expected):
     assert query_result == expected
 
 
-
-@pytest.mark.parametrize("querystring, expected", [('xx,', 'xx '),  # trailing space
+@pytest.mark.parametrize("querystring, expected", [('xx,', 'xx'),  # trailing space
```
Comment (halge) [inline tests/integration_tests/search/test_querybuilder.py:36]: trailing space in comment?

File: tests/integration_tests/search/test_querybuilder.py [hunk 3]
```diff
@@ -34,29 +33,28 @@ def test_build_occupation_collection_query(collection_id, expected):
...
+                                                   (' xx', 'xx'),
                                                    ('x x', 'x x'),
-                                                   ('x,x ', 'x x '),  # trailing space
-                                                   ('x.x ', 'x.x '),
-                                                   ('x!x ', 'x!x '),
-                                                   ('x?x ', 'x?x '),
-                                                   ('x:x ', 'x:x '),
-                                                   ('x;x ', 'x;x '),
-                                                   ('xx ', 'xx '),
+                                                   ('x,x ', 'x x'),  # trailing space
```
Comment (halge) [inline tests/integration_tests/search/test_querybuilder.py:51]: trailing space in comment?

File: tests/integration_tests/search/test_querybuilder.py (hunk 1 above)
Comment (oskardrenske-ith) [reply tests/integration_tests/search/test_querybuilder.py:43]: comments removed after behavoir changed

**Review by halge — APPROVED** [review]

**Review by lina1101 — APPROVED** [review]

**Pull request merged**